### PR TITLE
Image input

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
add: image comp
      add: input comp

### CODE REVIEW

File: components/image-input/image.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+          <div class="sub-heading">
+            You can add class <span class="highlight">responsive-img</span> to
+            make your image fit the width of container. It's height will get
+            adjusted by keeping the aspect ratio same. If you want to change the
+            aspect ratio, you will have to crop the image.
+          </div>
+          <div class="demo-component-typo">
+            <img
+              class="responsive-img"
+              src="/assets/res-demo-img.jpeg"
```
Comment (kirti2602) [inline components/image-input/image.html]: This image is taking a lot of time for me to load. It's more than 1 MB in size. So maybe try a lighter version

**Review by kirti2602 — APPROVED** [review]
Except for the suggested changes, the rest looks fine.

**Review by Asuraking1n — COMMENTED** [review]
UI awesome, responsiveness cool

File: components/image-input/image.html (hunk 1 above)
Comment (harshitpaliwal95) [reply components/image-input/image.html]: it is fixed now

**Pull request merged**